api/timeline/v1/timeseries.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package clutch.timeseries.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/timeseries/v1;timeseriesv1";
+
+import "google/protobuf/any.proto";
+import "validate/validate.proto";
+
+message TimeRange {
+  int64 start_time_millis = 1 [ (validate.rules).int64.gt = 0 ];
+  int64 end_time_millis = 2 [ (validate.rules).int64.gt = 0 ];
+}
+
+// A timeseries Point message is useful for organizing events
+// to be displayed in a timeline view.  Users can transform data
+// into a timeseries format and be able to organize them.
+message Point {
+  oneof event {
+    option (validate.required) = true;
+    TimeRange range = 2;
+    int64 singular_event_timestamp_millis = 3;
+  }
+  google.protobuf.Any event_info = 5;
+  string event_description = 6;
```

Comment:
```suggestion
  google.protobuf.Any pb = 4;
  string description = 5;
```

(to more closely match https://github.com/lyft/clutch/blob/498410f57fc1b534d941ec2473e1abf3e58d5609/api/topology/v1/topology_api.proto#L224)